### DISCUSSION TITLE
Mouse sensitivity on osu lazer seems different than stable

### DISCUSSION THREAD

**Discussion by lAg007** [ppy/osu#24583] [Q&A]
I use the mouse sensitivity at 1x and 800 dpi, it's a little fast, but I got used to it, when I went to test the laser it seems that the sensitivity was lowered, even with the same parameters.  

I use raw input enabled, and mouse acceleration disabled in windows.

another related problem is a certain input lag when moving the mouse, but I think it's just lag problems in the game itself. so I don't count in this discussion.

Setup: 
Processor: Intel Pentium E5700 Dual-Core
GPU: Integrated
Keyboard: Membrane, Fortrek Blackfire New
Mouse: Oex Combo Stage Mc.101

**Comment by peppy** [comment]
It should be the same, we have implemented it using the same implementation.

You could be running the games at different resolutions, or something on your system could have per-application profiles (ie. your mouse software).
